spec/rubocop/cop/lint/safe_navigation_consistency_spec.rb

```diff
@@ -15,44 +21,74 @@
     RUBY
   end
 
+  it 'allows || with comparison method' do
+    expect_no_offenses(<<~RUBY)
+      foo == bar || foo == baz
+    RUBY
+  end
+
   it 'allows safe navigation when different variables are used' do
     expect_no_offenses(<<~RUBY)
       foo&.bar || foobar.baz
     RUBY
   end
 
-  it 'allows calls to methods that nil responds to' do
+  it 'allows an offense using calling `nil?` after safe navigation with `||`' do
```

Comment:
For this test and the next one it'd make more sense to say "does not register an offense" instead of "allows an offense" IMO